paper/paper.md

```diff
@@ -117,8 +117,8 @@ $$\min_{\beta_0, \beta} \frac{1}{N} \sum_{i = 1}^N \mathcal{L} (y_i, \beta_0 + \
 + \lambda \mathcal{P}(\beta)$$
 
 where $\mathcal{L} (y_i, \beta_0 + \beta^T x_i)$ is the negative log-likelihood of an
-observation $i$, and \lambda $\mathcal{P}(\cdot)$ is the penalty that regularizes the solution, 
-where \lambda is a constant that controls the amount of regularization.
+observation $x_i$, and $\lambda \mathcal{P}(\cdot)$ is the penalty that regularizes the solution,
```

Comment:
@jasmainak @pavanramkumar  `$i$`, $(x_i, y_i)$ or even $y_i$ seem more accurately descriptions of an observation no?

Reply:
agreed, updated to reflect this!